ugvc/pipelines/mrd/featuremap_to_dataframe.py

```diff
@@ -43,52 +43,29 @@ def __parse_args(argv: list[str]) -> argparse.Namespace:
         "-if",
         "--info_fields",
         type=str,
-        nargs="+",
         required=False,
-        default="all",
-        help="""List of input info fields to include in dataframe, by default 'all''
-        If 'all' then all the info fields are read to columns
-        If None then no info fields are read to columns""",
+        default=None,
+        help="""Comma-separated list of input info fields to include in dataframe,
+        If None (default) then all the info fields are read to columns""",
```

Comment:
Why this change? default=None is good, but no reason for the comma separated list, with nargs='+' you can do:
--info_fields field1 field2 field3 
etc.